src/components/layouts/ComicsLayout/ComicsLayout.js

```diff
@@ -0,0 +1,91 @@
+import { useNavigation } from "@react-navigation/native";
+import React from "react";
+import {
+  View,
+  ActivityIndicator,
+  StyleSheet,
+  TouchableOpacity,
+} from "react-native";
+import { FlatList } from "react-native-gesture-handler";
+
+import Header from "../../../components/Header";
+import useMarvelData from "../../../hooks/ApiHelperHook";
+import theme from "../../../styles/theme/theme";
+import ComicCard from "../../Cards/ComicCard";
+import routes from "../../../navigation/routes";
+import StatusIndicator from "../../StatusIndicator";
+import { useTranslation } from "react-i18next";
+
+export default function ComicsLayout({ endpoint }) {
+  const { data: comics, error, isLoading } = useMarvelData(endpoint);
```

Comment:
Neden layout katmanında data çekiliyor? Layout yapıları zaten bunu yapmamak için var. Üst katmanda çekip buraya sadece data atmalıydınız.